account_credit_control/views/credit_control_line.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="credit_control_line_form" model="ir.ui.view">
+        <field name="name">credit.control.line.form</field>
+        <field name="model">credit.control.line</field>
+        <field name="arch" type="xml">
+            <form string="Control Credit Lines">
+                <header>
+                    <field name="state" widget="statusbar"
+                           statusbar_visible="draft,to_be_sent,sent"
+                           statusbar_colors='{"error":"red","email_error":"red"}'/>
+                </header>
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="partner_id"/>
+                            <field name="policy_level_id"/>
+                            <field name="policy_id"/>
+                            <field name="level"/>
+                            <field name="manually_overridden"/>
+                        </group>
+                        <group>
+                            <field name="date"/>
+                            <field name="date_due"/>
+                            <field name="channel"/>
+                            <field name="date_sent"/>
+                            <field name="mail_message_id"/>
+                        </group>
+                    </group>
+                    <group>
+                        <group>
+                            <field name="invoice_id"/>
+                            <field name="move_line_id"/>
+                            <field name="account_id"/>
+                        </group>
+                        <group>
+                            <field name="amount_due"/>
+                            <field name="balance_due"/>
+                            <field name="currency_id"/>
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="credit_control_line_search" model="ir.ui.view">
+        <field name="name">Credit Control Lines</field>
+        <field name="model">credit.control.line</field>
+        <field name="arch" type="xml">
+            <search string="Control Credit Lines">
+                <group string="Filters">
```

Comment:
Add name selector if you can, as it is recommended at https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0